File: SnappetChallenge/Models/Answer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace SnappetChallenge.Json
{
    public class Answer
    {
        public string SubmittedAnswerId;
        public DateTime SubmitDateTime;
        public int Correct;
        public string Progress;
        public string UserId;
        public string ExerciseId;
        public string Difficulty; 
        public string Subject;
        public string Domain;
        public string LearningObjective;
    }
}

File: TeacherDashboard/src/app/interfaces/answer.ts
export interface Answer {
  SubmittedAnswerId: number;
  SubmitDateTime: Date;
  Correct: boolean;
  Progress: number;
  UserId: number;
  ExerciseId: number;
  Difficulty: string;
  Subject: string;
  Domain: string;
  LearningObjective: string;

}
